src/test/regress/sql/multi_mx_router_planner.sql

```diff
@@ -10,6 +10,11 @@
 
 \c - - - :worker_1_port
 -- this table is used in a CTE test
+BEGIN;
+  SET local client_min_messages TO WARNING;
+  DROP TABLE IF EXISTS authors_hash_mx;
+COMMIT;
```

Comment:
Let's drop this at the end if possible. And otherwise add a comment why it cannot be done at the end.